kin-recovery/src/main/java/kin/recovery/RestoreFinishCallback.java

```diff
@@ -0,0 +1,7 @@
+package kin.recovery;
+
+public interface RestoreFinishCallback extends RestoreCallback {
```

Comment:
can it be internal?
also do not inherit from `RestoreCallback`, so you won't get the redundant `onSuccess`.
or maybe have generic `Callback<T>`, then you'll have once in type of string and once of type of `KinAccount`

Reply:
Of course it can be internal and i will change it.
Regarding the inheritance, indeed it kind of a workaround in order not to make things complicate and also they have a connection between them.
I will check how I can design it better.